plugins/modules/monitoring/sensu/sensu_check.py

```diff
@@ -272,7 +272,7 @@ def sensu_check(module, path, name, state='present', backup=False):
                     changed = True
                     reasons.append('`custom param {opt}\' was added'.format(opt=k))
                 check[k] = v
-            simple_opts += custom_params.keys()
+            simple_opts += list(custom_params.keys())
```

Comment:
```suggestion
            simple_opts += custom_params.keys()
```

Reply:
Not sure that `list` + `dict_keys` work

Reply:
In fact it doesn't:
```python
>>> a = dict(aa=1, ab=2, ac=3)
>>> [] + a.keys()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate list (not "dict_keys") to list
```

Reply:
It does:
```
>>> a = [1]
>>> b = {2: 3}
>>> a += b.keys()
>>> a
[1, 2]
```

Reply:
(That only works with `+=` though, not with `+`.)

Reply:
Please note that you tested `+`, while the code has `+=`.

Reply:
oooohhhhh shoot... why on earth do they behave differently? fixing...